R/board_s3.R

```diff
@@ -41,13 +41,15 @@
 #'
 #' # Details
 #'
-#' * If you point at a bucket that's not created by pins, some functions
-#'   like `pins_list()` will work, but won't return useful output.
+#' * The functions in pins do not create a new bucket. You can create
+#'   a new bucket from R with [paws][paws.storage::s3_create_bucket].
+#' * Some functions like `pins_list()` will work for an S3 board, but don't
```

Comment:
Should pins_list be pin_list? (super tiny nit!)

Reply:
Thanks! And how long has that typo been there, I wonder??? 🙈